<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="welcomeL">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pokemon Multiplayer Battle</string>
    </property>
   </widget>
   <widget class="QLabel" name="ql">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose 1 pokemon:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipL">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address : </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CharmandarRB">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Charmander</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="PikachuRB">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>191</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pikachu</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>97</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="BulbasaurRB">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Bulbasaur</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="ipAddress">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>97</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="PidgeotRB">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pidgeot</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="SquirtleRB">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Squirtle</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="GeodudeRB">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Geodude</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="player1BG"/>
 </buttongroups>
</ui>
